assets/cilium-k8s-perf/queries.libsonnet

```diff
@@ -0,0 +1,364 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local variables = import './variables.libsonnet';
+local prometheus = g.query.prometheus;
+
+{
+    ciliumControllerFailures: {
+        query():
+            prometheus.withExpr('cilium_controllers_failing')
+            + prometheus.withFormat('time_series')
+            + prometheus.withIntervalFactor(2)
+            + prometheus.withLegendFormat('{{ instance }} - {{ pod }}')
+            + prometheus.withDatasource('$Datasource')
+    },
+
+    ciliumIPAddressAllocation: {
+        query():
+           prometheus.withExpr('cilium_ip_addresses')
+            + prometheus.withFormat('time_series')
+            + prometheus.withIntervalFactor(2)
+            + prometheus.withLegendFormat('{{ pod }} - {{ family }}')
+            + prometheus.withDatasource('$Datasource') 
+    },
+
+    ciliumContainerCPU: {
+        query():
+            prometheus.withExpr('sum(irate(container_cpu_usage_seconds_total{container=~\"cilium.*\",container!=\"cilium-operator.*\",namespace!=\"\"}[$interval])) by (instance,pod,container,namespace,name,service) * 100')
+            + prometheus.withFormat('time_series')
+            + prometheus.withIntervalFactor(2)
+            + prometheus.withLegendFormat('{{ instance }} - {{ pod }}')
+            + prometheus.withDatasource('$Datasource') 
+    },
+
+    ciliumConatinerMemory: {
+        query():
+            prometheus.withExpr('container_memory_rss{container=~\"cilium.*\",namespace!=\"\"}')
```

Comment:
Same comment as above, just look at the namespace

Reply:
sure , i will do that.